MAPL_Base/MAPL_ExtDataGridCompMod.F90

```diff
@@ -2788,12 +2798,13 @@ subroutine OffsetTimeYear(inTime,yrOffset,outTime,rc)
 
      end subroutine OffsetTimeYear
 
-     subroutine GetBracketTimeOnSingleFile(fdata,tSeries,cTime,bSide,UniFileClim,interpTime,fileTime,tindex,allowExtrap,climyear,rc)
+     subroutine GetBracketTimeOnSingleFile(fdata,tSeries,cTime,bSide,UniFileClim,UniFileDOW,interpTime,fileTime,tindex,allowExtrap,climyear,rc)
         class(FileMetadataUtils),            intent(inout) :: fdata
         type(ESMF_Time),                     intent(in   ) :: tSeries(:)
         type(ESMF_Time),                     intent(inout) :: cTime
         character(len=1),                    intent(in   ) :: bSide
         logical,                             intent(in   ) :: UniFileClim
+        logical,                             intent(in   ) :: UniFileDOW
```

Comment:
Should this argument be optional (and then moved to the end of the list)? To preserve backward compatibility.

Reply:
And the PR does not appear to have a corresponding change in the caller.   Not an expert in this layer - just a meta comment about something that seems odd.